net/suricata/Config.in

```diff
@@ -0,0 +1,36 @@
+menu "Configuration"
+	depends on PACKAGE_suricata
+
+config SURICATA_LOGROTATE
+	bool "Copy Suricata logrotate file"
+	default n
+
+config SURICATA_DEBUG
+	bool "Enable debug output"
+	default n
+
+config SURICATA_LUA
+	bool "Enable Lua support"
+	default n
```

Comment:
why would you have this off by default?  Lua is very often installed, being used by luci, and being one of the only scripting languages easily runnable on openwrt platforms.